db/seed.sql

```diff
@@ -0,0 +1,54 @@
+INSERT INTO salz.users (username, icon_path, identity_provider, identity_provider_id) VALUES 
+    ('testuser1', 'user1.png', 'fakeprovider', 'fakeid1'),
+    ('testuser2', 'user2.png', 'fakeprovider', 'fakeid2'),
+    ('testuser3', 'user3.png', 'fakeprovider', 'fakeid3'),
+    ('testuser4', 'user4.png', 'fakeprovider', 'fakeid4'),
+    ('testuser5', 'user5.png', 'fakeprovider', 'fakeid5'),
+    ('testuser6', 'user6.png', 'fakeprovider', 'fakeid6'),
+    ('testuser7', 'user7.png', 'fakeprovider', 'fakeid7'),
+    ('testuser8', 'user8.png', 'fakeprovider', 'fakeid8'),
+    ('testuser9', 'user9.png', 'fakeprovider', 'fakeid9'),
+    ('testuser10', 'user10.png', 'fakeprovider', 'fakeid10');
+
+-- Create a bot for each user
+INSERT INTO salz.bots (user_id, upload_path, status)
+SELECT id, 'bots/' || id || '.zip', 'healthy' FROM salz.users;
+
+-- Create a bunch of games for each user
+INSERT INTO salz.games (status)
+SELECT 'finished' FROM generate_series(1, 50);
```

Comment:
note that most of the other columns in `salz.games` have a default value, so we just need `status`.
`generate_series` returns a table with a number from  1 to 50 ... in this case, it's more or less used as a dumb for loop (`select 'finished'` for each row in generate_series(1,50)`).  so the end result is that we create 50 games all with the status 'finished' (rest of the columns are default values)